packages/survey-vue3-ui/src/QuestionComment.vue

```diff
@@ -11,8 +11,8 @@
     :value="question.comment"
     :maxlength="question.getOthersMaxLength()"
     :placeholder="question.renderedCommentPlaceholder"
-    :aria-label="question.ariaLabel"
-    :aria-required="question.ariaRequired"
+    :aria-label="question.ariaLabel || question.a11y_input_ariaLabel"
```

Comment:
question.ariaLabel || question.a11y_input_ariaLabel
It looks like they always should be the same.